src/Miunie.Discord/Services/HelpService.cs

```diff
@@ -0,0 +1,129 @@
+﻿// This file is part of Miunie.
+//
+//  Miunie is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  Miunie is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with Miunie. If not, see <https://www.gnu.org/licenses/>.
+
+using Discord.Commands;
+using Discord.WebSocket;
+using Miunie.Core.Entities;
+using Miunie.Core.Providers;
+using Miunie.Discord.Attributes;
+using Miunie.Discord.Embeds;
+using Miunie.Discord.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Miunie.Discord
+{
+    public class HelpService
+    {
+        private readonly CommandService _commandService;
+        private readonly ILanguageProvider _lang;
+
+        public HelpService(CommandService commandService, ILanguageProvider lang)
+        {
+            _commandService = commandService;
+            _lang = lang;
+        }
+
+        public HelpResult GetDefault()
+        {
+            HelpResult entry = new HelpResult
```

Comment:
```suggestion
            var entry = new HelpResult
```

Use implicit declaration where obvious.